x/dispensation/keeper/executors_test.go

```diff
@@ -30,8 +30,8 @@ func TestKeeper_AccumulateDrops(t *testing.T) {
 func TestKeeper_CreateAndDistributeDrops(t *testing.T) {
 	app, ctx := test.CreateTestApp(false)
 	keeper := app.DispensationKeeper
-	inputList := test.GenerateInputList("15000000000000000000")
-	outputList := test.GenerateOutputList("10000000000000000000")
+	inputList := test.CreatInputList(3, "15000000000000000000")
```

Comment:
`CreatInputList` should probably be `CreateInputList` - could be worth fixing the typo and then updating tests with the correct function name